alerts.md

```diff
@@ -16,6 +16,10 @@ Kubecost alerts allow teams to receive updates on real-time Kubernetes spend. Th
  
 6. [Cluster Health](#type-cluster-health) -- used to determine if the cluster's health score changes by a specific threshold.
 
+7. [Asset Budget](#type-asset-budget) -- sends an email and/or Slack alert when a particular asset spend crosses a defined threshold.
```

Comment:
How about "when spend for a particular set of assets crosses a defined threshold."